packages/spark-core/components/_narrow-navigation.scss

```diff
@@ -0,0 +1,56 @@
+// ==================================================================
+// Narrow Navigation
+// ==================================================================
+.sprk-c-NarrowNavigation {
+  background-color: $white;
+  border-top: 2px solid $gray-medium;
+  list-style-type: none;
+  position: absolute;
+  width: 100%;
+}
+
+.sprk-c-NarrowNavigation__item:last-child .sprk-c-NarrowNavigation--sub {
+  border-bottom: 0;
+}
+
+.sprk-c-NarrowNavigation--sub {
+  border-bottom: 2px solid $gray-medium;
+  border-top: 0;
+  position: static;
+}
+
+.sprk-c-NarrowNavigation__link {
+  border: 2px solid $gray-medium;
+  border-top: 0;
+  color: $blue-dark;
+  display: block;
+  padding: $space-inset-m;
+  text-decoration: none;
+  text-transform: uppercase;
+  width: 100%;
+
+  &:focus,
+  &:hover {
+    background-color: $blue-dark;
+    color: $white;
+
+    .sprk-c-NarrowNavigation__arrow {
+      fill: $white;
```

Comment:
The spark icons scss has the fill set to currentColor. Is that getting overridden here?